libs/payments/cart/src/lib/cart.manager.ts

```diff
@@ -101,6 +101,7 @@ export class CartManager {
         ...cart,
         id: cart.id.toString('hex'),
         uid: cart.uid ? cart.uid.toString('hex') : undefined,
+        metricsOptedOut: null,
```

Comment:
I'd expect `metricsOptedOut` not to be part of `cart.manager` since its main purpose is to help manage the Cart database table.

Since `metricsOptedOut` isn't part of the DB table, and shouldn't be, I wouldn't expect any changes here.

Reply:
bumped to the service layer